jans-cedarling/cedarling_pg/Dockerfile

```
@@ -0,0 +1,112 @@
+FROM rust:${RUST_VERSION}-alpine${ALPINE_VERSION} AS builder
+
+ARG PG_MAJOR
+
+# pgrx + cedarling native build deps on Alpine.
+#  - build-base, musl-dev, linux-headers: musl toolchain
+#  - clang-dev, llvm-dev: bindgen runtime (pgrx-pg-sys)
+#  - pkgconf, perl: openssl-sys / native crate builds
+#  - openssl-dev: TLS dependencies
+#  - postgresql${PG_MAJOR}-dev: server headers + pg_config matching the runtime
+#  - bash: cargo pgrx init runs a bash script
+RUN apk add --no-cache \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)